cmd/pipe-main.go

```diff
@@ -89,23 +107,42 @@ EXAMPLES:
 `,
 }
 
-func pipe(targetURL string, encKeyDB map[string][]prefixSSEPair, storageClass string, meta map[string]string) *probe.Error {
+func pipe(ctx *cli.Context, targetURL string, encKeyDB map[string][]prefixSSEPair, meta map[string]string) *probe.Error {
 	if targetURL == "" {
 		// When no target is specified, pipe cat's stdin to stdout.
 		return catOut(os.Stdin, -1).Trace()
 	}
+
+	storageClass := ctx.String("storage-class")
 	alias, _ := url2Alias(targetURL)
 	sseKey := getSSE(targetURL, encKeyDB[alias])
 
+	multipartThreads := ctx.Int("concurrent")
```

Comment:
We should probably do [`debug.SetGCPercent(20)`](https://pkg.go.dev/runtime/debug#SetGCPercent) if `multipartThreads > 1` to reduce default GC overhead, since we are allocating BIG buffers.